source/privacy.html.erb

```diff
@@ -167,7 +167,7 @@ title: Privacy notice
         <ul class="govuk-list govuk-list--bullet">
           <li>
             <%=
-              link_to 'WorldPay Privacy Policy',
+              link_to 'Worldpay Privacy Policy',
```

Comment:
They are not consistent about their capitalisation, so I don't see why we should be!